packages/chakra-ui-core/src/CAvatar/CAvatar.js

```diff
@@ -250,15 +249,16 @@ const CAvatar = {
             h: '100%',
             rounded: 'full',
             objectFit: 'cover',
-            alt: this.name
+            alt: this.name,
+            src: this.src
           },
           domProps: {
             src: this.src
```

Comment:
this line is causing the issue. Comment out and run the tests and it works.

Reply:
Removed. This should have been removed. Good catch!